graylog2-web-interface/src/pages/EditEventNotificationPage.tsx

```diff
@@ -98,8 +98,8 @@ class EditEventDefinitionPage extends React.Component<
             path: DocsHelper.PAGES.ALERTS,
           }}>
           <span>
-            Notifications alert you of any configured Event when they occur. Graylog can send Notifications directly to
-            you or to other systems you use for that purpose.
+            Notifications alert you of any configured Event when they occur. Notifications can be send directly to you
```

Comment:
```suggestion
            Notifications alert you of any configured Event when they occur. Notifications can be sent directly to you
```

Also, this is duplicated. Do we want to extract a shared component?